Resources/Prototypes/Catalog/Fills/Boxes/security.yml

```diff
@@ -57,3 +57,35 @@
     layers:
       - state: box_security
       - state: ziptie
+
+- type: entity
+  name: box of beanbag shots
+  parent: BoxCardboard
+  id: BoxBeanbag
+  description: A box full of beanbag shots, designed for riot shotguns.
+  components:
+  - type: StorageFill
+    contents:
+      - id: ShellShotgunBeanbag
+        amount: 6
```

Comment:
I think there's a component for it, ammoboxcomponent?

Reply:
That's for the static, high capacity, deathmatchy, 'ammunition boxes' I believe. These are just like a normal box (i.e., can put them into hands, bags, whatever) but with shells in them.

Reply:
Really I care more about the entity count because the ammo box doesn't instantiate entities until necessary which is much better.

Reply:
Yes, but I do not think that is suitable for this item because a) it should have its inventory visible, b) it should be portable and c) it has a fraction of the entities of the ammo box (6 vs. 200). StorageFill works better in these circumstances.